mod/message.php

```diff
@@ -282,7 +282,6 @@ function message_content(App $a)
 			$messages = DBA::toArray($messages_stmt);
 
 			DBA::update('mail', ['seen' => 1], ['parent-uri' => $message['parent-uri'], 'uid' => local_user()]);
-			DBA::update('notify', ['seen' => 1], ['type' => Type::MAIL, 'parent' => $message['id'], 'uid' => local_user()]);
```

Comment:
Why did you remove this code?

Reply:
https://github.com/friendica/friendica/blob/818c064c0a5d75b25e1977d336e319e8e9464577/include/enotify.php#L146-L147

They hadn't been displayed anyway in the notifications:
https://github.com/friendica/friendica/blob/develop/mod/ping.php#L414-L424

Unread mails are visible without the use of the `notify` table. And also I had to remove the assigning of the `parent` value since this value isn't referring to the `item` table.